### ISSUE TITLE
[suggestion] Add real time network monitor to Dashboard

### ISSUE THREAD

**Issue by vampywiz17** [webmin/webmin#517]
I really miss a option to see the actual bandwith usage in webmin. I know, it possible to use the bandwidth monitor, but i think a good idea a real time monitor (based on iftop, or something else) on Dashboard.

**Comment by jcameron** [comment]
Nice idea, but collecting this would be fairly complex and so isn't likely to happen any time soon.

**Comment by swelljoe** [comment]
Ilia and I have been discussing ways to get realtime data into the dashboard, without causing an undue burden on the system for collection. We haven't yet sorted it out, but realtime stats are definitely on the radar for the next few months. Network will be a bit down the road, as we aren't already collecting that info (at least not realtime, or even close to it)...we'll start with the data we already have coming in regularly. But, network data isn't a bad idea.

**Comment by mlbarrow** [comment]
Apologize that I have not been following this thread super closely and if I
repeat something already suggested. Nevertheless, perhaps there can be some
inspiration in this project:

https://github.com/firehol/netdata


*michael at barrow dot me*
+1.541.600.2027

"Do not anticipate trouble, or
worry about what may never happen.
Keep in the sunlight." -- B. Franklin

**Comment by vampywiz17** [comment]
or check this:  https://github.com/ajenti/ajenti 

It use a simpe in/out number. It is enought.

**Comment by qooob** [comment]
@vampywiz17 Yes, we're aware of it. We will add it as soon as we sort upcoming issues, which one and  most important one is [_Mail_](https://github.com/qooob/authentic-theme/issues/431). Afterwards, we will take a look at statistics.
Do you find _Ajenti_ even remotely comparable/competitive with _Webmin/Virtualmin_?

**Comment by gnadelwartz** [comment]
I had a look at Ajenti hompage and was impressed about the features and screenshots, but when I tried to install it for testing I gave up after 2 hours of frustration.

Is there is anyone out there who got Ajenti installed and running or its only paperware?

**Comment by znhskzj** [comment]
Does anyone know how to change the "Realtime Monitoring" link on the bottom left of the Webmin Dashboard? The default format "hostname:19999" looks like it's for netdata.

**Comment by iliajie** [comment]
> Does anyone know how to change the "Realtime Monitoring" link on the bottom left of the Webmin Dashboard? The default format "hostname:19999" looks like it's for netdata.

Yes, you can change the default link by opening the browser's console and running:
```
settings_leftmenu_netdata_link = 'https://analytics.google.com';
theme.config.save();
```

**Comment by znhskzj** [comment]
Thank you for sharing this. I would like to confirm if this method only temporarily changes the corresponding link. Is it possible to make a permanent change on the server side by modifying configuration files such as “miniserv.conf”? I haven't found any configuration files that seem to be related in the “/etc/webmin” directory.

Sincerely
Rong Zhu
________________________________

**Comment by iliajie** [comment]
> I would like to confirm if this method only temporarily changes the corresponding link.

As calling `theme.config.save();` supposed to make this change permanent! The config file should be saved to `/etc/webmin/authentic-theme` directory.
